egs/sre08/v1/sid/nnet3/xvector/allocate_egs.py

```diff
@@ -0,0 +1,325 @@
+#!/usr/bin/env python
```

Comment:
If this script would be trivial to make a python3 script, and you can test it, it would be great to have this be python3.
I'm encouraging python3 for new scripts now; we have accepted the dependency.